.github/workflows/on_pr_push_commit.yml

```diff
@@ -0,0 +1,52 @@
+# ******************************************************************
+# This workflow will trigger once a PR opens to main
+# ******************************************************************
+
+name: SDK Testing
+
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  pull-requests: write
+  contents: write
+  issues: write
+
+jobs:
+  runUpdate:
+    name: setup
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Main branch checkout
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          ref: ${{ github.head_ref }}
+
+      # Setup .npmrc file to run npm update
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
```

Comment:
We should probably change this to 18.x node version like our other nodejs repos.

Reply:
done